dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -35,6 +37,8 @@ dom0-rpc-qubes.Filecopy:
     - marker_start: "### BEGIN securedrop-workstation ###"
     - marker_end: "### END securedrop-workstation ###"
     - content: |
+        sd-log @default ask
+        sd-log @tag:receive-sd-logs ask
```

Comment:
Let's make sure all the tags added by the SDW config begin with `sd-`, for consistency. So: `receive-sd-logs` -> `sd-receive-logs`, etc. Will make it easier over time to reason about collections and interactions between RPC policies.

Reply:
The requirement to include `@default ask` here is far from intuitive, but the citation you provided was informative, and functional testing locally does indeed line up with what's intended.

Reply:
(This is done.)